### ISSUE TITLE
2.0.0-beta8 Flickers/Blinks

### ISSUE THREAD

**Issue by hudiohizari** [googlecodelabs/constraint-layout#87]
Flicks to start state when first touch on anchor at end state for few ms, then back to end state, then continue animation normally.

`
    <Transition
        motion:constraintSetEnd="@id/end"
        motion:constraintSetStart="@id/start">
        <OnSwipe
            motion:onTouchUp="autoComplete"
            motion:dragDirection="dragUp"
            motion:touchAnchorId="@id/rvProducts" />
    </Transition>
`

**Comment by arberg** [comment]
Yeah, as mentioned in the auto-linked comment, I think this issue is related to my above, so I also think this issue is not there in 2.0.0-beta7.
